test/fixtures/cert-manager-values.yaml

```diff
@@ -13,7 +13,7 @@ resources:
     memory: 200Mi
 
 extraArgs:
-- --v=4
+- --v=10
```

Comment:
We may want to reduce this once we improve logging/debugging options in the e2e suite